cpp/inc/bond/core/reflection.h

```diff
@@ -101,24 +101,21 @@ struct FieldTemplate
     /// @brief Static data member describing field metadata
     static const Metadata& metadata;
 
-    /// @brief Static data member representing the field pointer
-    static const field_pointer field;
-
```

Comment:
This looks like a breaking change (removal of public member of public interface). Is it intentional? If so, what's motivating making this breaking change?

It appears that the replacement for this functionality is `&theFieldType.GetVariable(someStruct)`.